rollup.config.mjs

```diff
@@ -35,7 +37,7 @@ export default [
       peerDepsExternal(),
       resolve(),
       commonjs(),
-      babel({ exclude: 'node_modules/**' }),
+      babel({ exclude: 'node_modules/**', babelHelpers: 'bundled' }),
```

Comment:
Nothing new. It's basically the same config as before but it just removes one console warning, because now it should be explicitly declared.
<img width="730" alt="image" src="https://github.com/timberhubcom/timberhub-component-library/assets/53660145/d494782c-8847-4403-be17-5e7f73a153e1">